app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/select.phtml

```diff
@@ -12,49 +12,65 @@
 <?php $_skipSaleableCheck = $this->helper(Magento\Catalog\Helper\Product::class)->getSkipSaleableCheck(); ?>
 <?php list($_defaultQty, $_canChangeQty) = $block->getDefaultValues(); ?>
 
-<div class="field admin__field option<?php if ($_option->getDecoratedIsLast()) :?> last<?php endif; ?><?php if ($_option->getRequired()) { echo ' _required'; } ?>">
+<div class="field admin__field option<?php if ($_option->getDecoratedIsLast()): ?> last<?php endif; ?>
+<?php if ($_option->getRequired()): ?>
+    <?= ' _required' ?>
+<?php endif; ?>">
```

Comment:
II would say it's too much logic for a template can you use ViewModel to resolve this conditions?